src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -91,6 +110,17 @@
       <Link>Common\System\HResults.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <Compile Include="System\IO\FileStreamHelpers.CoreClr.cs" />
+    <Compile Include="System\IO\TextReader.netstandard17.cs" />
+    <Compile Include="System\IO\TextWriter.netstandard17.cs" />
+    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
+      <Project>{E7E8DE8A-9EC1-46A8-A6EE-727DB32DBEB8}</Project>
```

Comment:
Can you please group the ProjectReference with the other references in this project? I would like to keep them mostly clustered together.